src/modules/migrations/v090Migrations/migrateSenses.js

```diff
@@ -0,0 +1,34 @@
+export default function migrateSenses(actorData, updateData) {
+  const old = actorData?.system?.attributes?.senses;
+
+  const newSenses = Object.entries(old).reduce((acc, [sense, distance]) => {
+    const mileUnits = ['mile', 'miles', 'mi.', 'mi'];
+    const meterUnits = ['meter', 'meters', 'm.', 'm'];
+    const kilometerUnits = ['kilometer', 'kilometers', 'km.', 'km'];
+    const unlimitedUnits = ['unlimited', 'infinite'];
```

Comment:
Since these are going to be used over and over again, they should be defined outside of the `reduce` call. Being inside, every iteration of reduce causes these variables to be redefined.